DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/DeactivateAdmin.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Administrator
+@using Microsoft.Extensions.Configuration
+@model DeactivateAdminViewModel
+
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    ViewData["Title"] = errorHasOccurred ? "Error: Deactivate Admin Account" : "Deactivate Admin Account";
+    ViewData["Application"] = "Tracking System";
+    ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+    ViewData["HeaderPathName"] = "Tracking System";
+    var confirmError = ViewData.ModelState[nameof(DeactivateAdminViewModel.Confirm)]?.Errors?.Count > 0;
+    var confirmFormErrorClass = confirmError ? "nhsuk-form-group--error" : "";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred)
+    {
+      <vc:error-summary order-of-property-names="@new []{ nameof(DeactivateAdminViewModel.Confirm) }" />
+    }
+
+    <h1 class="nhsuk-heading-l">Are you sure you would like to deactivate this admin account?</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-s">
+    <div class="nhsuk-u-font-weight-bold">
+      User
+    </div>
+  </div>
+  <div class="nhsuk-grid-column-three-quarters nhsuk-heading-s nhsuk-u-font-weight-normal">
+    @Model.FullName (@Model.EmailAddress)
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form class="nhsuk-u-margin-bottom-5" method="post" asp-action="DeactivateAdmin">
+      <div class="nhsuk-form-group @confirmFormErrorClass">
+        <fieldset class="nhsuk-fieldset" aria-describedby="confirm-error">
```

Comment:
We have a bunch of ViewComponents for commonly used NHS form components that might save effort when using forms. In this case we have the CheckboxesViewComponent, which typically takes in a list of checkboxes, but it should work fine with just one. It would be good to use that view component here.

For an example have a look at the EditRolesViewModel and corresponding page to see how that implements it.

Reply:
Used checkboxes view component as suggested.